Marlin/SanityCheck.h

```diff
@@ -321,16 +321,21 @@
     #error "EXTRUDERS must be 1 with HEATERS_PARALLEL."
   #endif
 
-#elif ENABLED(SINGLENOZZLE)
+#elif ENABLED(SINGLENOZZLE) 
```

Comment:
Careful about trailing white-space. A regex search on `[ \t]+$` will locate all instances of trailing whitespace.